react/__tests__/CategoryMenu.test.js

```diff
@@ -1,64 +1,44 @@
 import React from 'react'
-import { mount } from 'enzyme'
-import { IntlProvider } from 'react-intl'
-import { CategoryMenuWithIntl } from '../index'
+import { render } from 'test-utils'
+import { LocalCategoryMenu } from '../index'
 
 describe('CategoryMenu component', () => {
-  let wrapper
+  const mockedCategories = [
```

Comment:
You could put this in a file in '__mocks__' folder.

Reply:
I think the __mocks__ folder isn't adequate to that, since its purpose is to provide mocks for required modules. The ideal would be to test using Apollo's MockedProvider, but I couldn't use it. Will try again in the future, but for while I think this is ok.

Reply:
since its a plain file, with no real logic envolved, I agree with Juan at this.

Reply:
Yes, the real issue would be to have an import from __mocks__, since that folder is destined to manual mocks of the modules used by the app being tested. Right now I don't think we have a folder for that type of file or a guideline for this situation, but I think we'll need that soon.